properties_pane/entity_level/entityLevelConfig.json

```diff
@@ -206,22 +198,17 @@ making sure that you maintain a proper JSON format.
 						"isCompositeKey": true,
 						"setPrimaryKey": false,
 						"template": "collectiontree",
-						"abbr": "PK",
-						"dependency": {
-							"key": "partitionBy",
-							"value": "keys"
-						}
+						"abbr": "PK"
 					},
 					{
 						"propertyName": "Expression",
 						"propertyKeyword": "partitioning_expression",
 						"propertyTooltip": "A SQL expression that returns a value from each row. In the simplest cases, it is a column name. This value is used to determine which partition should contain a row.",
 						"propertyType": "details",
-						"template": "textarea",
+						"template": "codeEditor",
```

Comment:
As I understand, Cockroach doesn't support partition by expression. So, why do you need this one?

Reply:
They are, CockroachDB requires both keys and an expression. It's also possible to create complex partitions with subpartitions: https://www.cockroachlabs.com/docs/v23.1/partitioning#step-4-create-a-table-with-the-appropriate-partitions